### PR TITLE
[docker-fpm-frr]: Generate separated staticd.conf for staticd

### PR DESCRIPTION
**Pull request by shikenghua**
Staticd was separated out from zebra after frr-6.0 to isolate static route handling. 
If staticd.conf is missing, it will take zebra.conf for backward compatible. In this case, many ERR logs will be generated because it doesn't recognize other CLI commands in zebra.conf except those for static route.
This may cause some Ansible tests fail because of the loganalyzer found errors in syslog.

**- What I did**
Generate separated staticd.conf for staticd

**- How I did it**
Generate staticd.conf by templates/staticd.conf.j2 with config DB data

**- How to verify it**
Add MGMT_INTERFACE with gwaddr in config_db.json. After reboot, check whether the staticd.conf is generated in /etc/frr/ with correct content and no staticd ERR message should be found in syslog.

**- Description for the changelog**
Generate staticd.conf by templates/staticd.conf.j2 with config DB data

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by pavel-shirshov** [comment]
retest this please

File: dockers/docker-fpm-frr/staticd.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+! file: staticd.conf
+!
+{% endblock banner %}
+!
+{% block sys_init %}
+hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
+password zebra
+enable password zebra
+{% endblock sys_init %}
+!
```
Comment (pavel-shirshov) [inline dockers/docker-fpm-frr/staticd.conf.j2]: I didn't find anything in staticd which could parse hostname, password and enable password options. Do we need them here?

File: dockers/docker-fpm-frr/staticd.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ip route 0.0.0.0/0 {{ MGMT_INTERFACE[(name, prefix)]['gwaddr'] }} 200
+{% endif %}
+{% endfor %}
+{% endblock default_route %}
+!
+{% block logging %}
+log syslog informational
+log facility local4
+{% endblock logging %}
+!
```
Comment (pavel-shirshov) [inline dockers/docker-fpm-frr/staticd.conf.j2]: The same for block logging. 
Is it supported by staticd?

**Comment by pavel-shirshov** [comment]
Also. If you introduce `ip routes` to the staticd.conf, should we remove them from zebra.conf?

File: dockers/docker-fpm-frr/staticd.conf.j2 (hunk 1 above)
Comment (shikenghua) [reply dockers/docker-fpm-frr/staticd.conf.j2]: I referred to the conf files generated by "copy running-config startup-config" in vtysh. hostname, password and logging options appear in all of the bgpd.conf, zebra.conf and staticd.conf, but enable password option only appears in zebra.conf and staticd.conf. 
> root@sonic:/# cat /etc/frr/staticd.conf
!
! Zebra configuration saved from vty
!   2019/08/09 09:53:10
!
frr version 7.0.1-sonic
frr defaults traditional
!
hostname sonic
password zebra
enable password zebra
log syslog informational
log facility local4
!
!
!
ip route 0.0.0.0/0 10.11.0.1 200
line vty
!

File: dockers/docker-fpm-frr/staticd.conf.j2 (hunk 2 above)
Comment (shikenghua) [reply dockers/docker-fpm-frr/staticd.conf.j2]: Same as the comment above. By the way, I think the logging block should be moved to the place below enable password option for consistency with the original conf files generated by FRR. And zebra.conf.j2 also need to do the same modification. What do you think?

**Comment by shikenghua** [comment]
yes, you are right. `ip routes` block should be removed from zebra.conf. Thanks.

File: dockers/docker-fpm-frr/staticd.conf.j2 (hunk 2 above)
Comment (pavel-shirshov) [reply dockers/docker-fpm-frr/staticd.conf.j2]: I think let's keep the options in the current order. Thank you for your comments.

**Comment by pavel-shirshov** [comment]
Can you please remove ip route and ipv6 route from zebra.conf and I'll push this change?

**Comment by shikenghua** [comment]
Done. Please review again.

**Comment by lguohan** [comment]
retest this please

**Comment by pavel-shirshov** [comment]
retest this please

**Comment by lguohan** [comment]
tests are failing, please fix the test.

**Comment by shikenghua** [comment]
Don't understand why the tests fail. What is "Build finished. No test results found" means? I checked other PRs and found that some of them passed with the same "Build finished. No test results found" messages. Maybe I just run the test again.

**Comment by shikenghua** [comment]
retest this please

**Comment by pavel-shirshov** [comment]
@shikenghua
Check tthe test
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/tests/test_j2files.py
This file should be fixed.
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/tests/sample_output/zebra_frr.conf
New sample for staticd should be introduced.
Check

**Comment by lguohan** [comment]
retest vs please

**Comment by pavel-shirshov** [comment]
retest baseimage please

**Pull request merged**